e2e-cypress/Jenkinsfile.template

```diff
@@ -49,27 +60,32 @@ odsComponentPipeline(
 
 def stageTest(def context) {
   stage('Integration Test') {
-    // OPTIONAL: load environment variables for Azure SSO with MSALv2; please adapt variable names to your OpenShift config
+    // Define your DEV and QA base URLs in a config map in OpenShift; please adapt variable names to your OpenShift config
```

Comment:
@roicarrera why define the DEV and QA base urls in a secret if 12-factor apps teaches us to provide them through the environment? I believe that ODS already does that. Then, a test case should only reference a path and resouce below the BASE URL. Can you clarify the design? What about PROD?